docs/UserGuide.md

```diff
@@ -1335,7 +1334,7 @@ Format: `createsession s/SESSION_NUMBER n/STUDENT_NAME n/STUDENT_NAME …`
 
 #### 🛠️Updating a Session's Remark: `updatesessionremark`
 
-You can update an existing session's remark.
+You can update an existing session's remark if there are changes or additional information that you want to include in the session notes or details.
```

Comment:
You can update an existing session's remark if you need to change or add additional information to the session notes or details.

Change it such that the format follows `if you need to ...`